crates/solidity/outputs/cargo/tests/src/doc_examples/using_queries.rs

```diff
@@ -0,0 +1,176 @@
+use std::path::Path;
+
+use anyhow::Result;
+use infra_utils::paths::PathExtensions;
+use semver::Version;
+use slang_solidity::kinds::RuleKind;
+use slang_solidity::language::Language;
+use slang_solidity::parse_output::ParseOutput;
+use slang_solidity::query::Query;
+
+fn parse_doc_input_file<T: AsRef<Path>>(path: T) -> Result<ParseOutput> {
+    let input_path =
+        Path::repo_path(Path::new("documentation/public/user-guide/inputs").join(path.as_ref()));
```

Comment:
nit suggestion: no need for the extra call:

```suggestion
        Path::repo_path("documentation/public/user-guide/inputs").join(path.as_ref());
```

Reply:
Makes sense, done